### PR TITLE
Added reactive wrapper for NSSegmentedControl selectedSegment property

### PR DESCRIPTION
**Pull request by plzen**
This PR added reactive wrapper for selectedSegment property of NSSegmentedControl

### CODE REVIEW

**Comment by plzen** [comment]
Hello @kzaher,

I see issues with checks on CI. Could you help me what I need to do to fix it?

**Comment by kzaher** [comment]
Hi @plzen , 

CI console says it all. 
```
Please run ./scripts/update-jazzy-config.rb`
```

File: RxCocoa/macOS/NSSegmentedControl+Rx.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    
+extension Reactive where Base: NSSegmentedControl {
+    
+    /// Reactive wrapper for `selectedSegment` property.
+    public var selectedSegment: ControlProperty<Int> {
+        return NSControl.rx.value(
+            base,
+            getter: { control in
+                return control.selectedSegment
+        },
```
Comment (kzaher) [inline RxCocoa/macOS/NSSegmentedControl+Rx.swift]: Indentation.

File: Tests/RxCocoaTests/NSSegmentedControl+RxTests.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+//  Copyright © 2017 Krunoslav Zaher. All rights reserved.
+//
+
+import RxSwift
+import RxCocoa
+import AppKit
+import XCTest
+
+final class NSSegmentedControlTests: RxTest {
+
```
Comment (freak4pc) [inline Tests/RxCocoaTests/NSSegmentedControl+RxTests.swift]: can you add more tests please?

Comment (plzen) [reply Tests/RxCocoaTests/NSSegmentedControl+RxTests.swift]: @freak4pc I've added additional tests for getting and setting selected segment

**Pull request closed without merging**